plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/dlf/TestingAlibabaDlfUtils.java

```diff
@@ -0,0 +1,257 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.hive.metastore.dlf;
+
+import com.aliyun.datalake.metastore.common.CacheDataLakeMetaStoreConfig;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import io.trino.plugin.hive.HdfsConfig;
+import io.trino.plugin.hive.HdfsConfiguration;
+import io.trino.plugin.hive.HdfsConfigurationInitializer;
+import io.trino.plugin.hive.HdfsEnvironment;
+import io.trino.plugin.hive.HiveBucketProperty;
+import io.trino.plugin.hive.HiveHdfsConfiguration;
+import io.trino.plugin.hive.HiveStorageFormat;
+import io.trino.plugin.hive.HiveType;
+import io.trino.plugin.hive.authentication.NoHdfsAuthentication;
+import io.trino.plugin.hive.metastore.Column;
+import io.trino.plugin.hive.metastore.Database;
+import io.trino.plugin.hive.metastore.HiveMetastore;
+import io.trino.plugin.hive.metastore.Partition;
+import io.trino.plugin.hive.metastore.StorageFormat;
+import io.trino.plugin.hive.metastore.Table;
+import io.trino.spi.security.PrincipalType;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.hive.metastore.api.StorageDescriptor;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Properties;
+import java.util.stream.Collectors;
+
+import static java.util.Objects.requireNonNull;
+
+public class TestingAlibabaDlfUtils
+{
+    private static final String WAREHOUSE_BASE_PATH = "file://" + System.getProperty("user.dir") + "/metastore-ut/";
+    private static final String WAREHOUSE_PATH = WAREHOUSE_BASE_PATH + "hive/warehouse/";
+    public static final String TEST_DB = "metastore_ut_test_db";
+    public static final String TEST_TABLE = "metastore_ut_test_tb";
+    private static final String CONFIG_FILE = "alibaba-dlf.properties";
+    private static final String DEFAULT_TABLE_TYPE = "MANAGED_TABLE";
+    private static final String DEFAULT_OWNER = "Mr.EMR";
+    private static final PrincipalType DEFAULT_OWNER_TYPE = PrincipalType.USER;
+    private static final Map<String, String> DEFAULT_PARAMETERS = ImmutableMap.of("key", "value");
+
+    private TestingAlibabaDlfUtils() {}
+
+    static HiveMetastore getDlfClient() throws IOException
+    {
+        HdfsConfig hdfsConfig = new HdfsConfig();
+        HdfsConfiguration hdfsConfiguration =
+                new HiveHdfsConfiguration(new HdfsConfigurationInitializer(hdfsConfig), ImmutableSet.of());
+        HdfsEnvironment hdfsEnvironment =
+                new HdfsEnvironment(hdfsConfiguration, hdfsConfig, new NoHdfsAuthentication());
+
+        Properties properties = new Properties();
```

Comment:
Rather than loading a resource file, simply call the setters on the config class:
```java
AlibabaDlfMetaStoreConfig dlfConfig = new AlibabaDlfMetaStoreConfig()
        .setEndpoint("datalake-daily.aliyuncs.com")
        .setAkMode(AKMode.MANUAL)
        ...;